docs/cookbook/django.rst

```diff
@@ -0,0 +1,140 @@
+
+Integrate Django (or Django Rest Framework)
+===========================================
+
+This cookbook will use DRF (Django REST Framework) as an example, but the syntax is exaclty
```

Comment:
sorry for bothering, "exaclty" should be fixed. I take the opportunity to thank you for this docs. If it'd be possible to handle [this case too](https://github.com/Miksus/rocketry/issues/76#issuecomment-1302731300) it would be great.

Reply:
Glad to have your opinion. If you like, have a look at this short code explaining the use case (which, IMHO, can be pretty common even if not appropriate for the get-started cookbook) https://github.com/paolodina/rocketry-django-usecase